baselines/dom.generated.d.ts

```diff
@@ -14262,6 +14284,188 @@ declare var HTMLSummaryElement: {
     new(): HTMLSummaryElement;
 };
 
+interface EXT_blend_minmax {
+    readonly MIN_EXT: number;
+    readonly MAX_EXT: number;
+}
+
+declare var EXT_blend_minmax: {
```

Comment:
not needed

Reply:
`EXT_blend_minmax` is declared as `[NoInterfaceObject]`, so the interface declaration itself is needed but `declare var` part is not needed.